Makefile

```diff
@@ -178,31 +178,34 @@ cfn/lint: | guard/program/cfn-lint
 ## Runs eclint against the project
 eclint/lint: | guard/program/eclint guard/program/git
 eclint/lint: HAS_UNTRACKED_CHANGES ?= $(shell cd $(PROJECT_ROOT) && git status -s)
-eclint/lint: ECLINT_FILES ?= git ls-files -z
+eclint/lint: ECLINT_FILES ?= $(shell git -C $(PROJECT_ROOT) ls-files -z | grep -zv ".bats" | xargs -0)
 eclint/lint:
 	@ echo "[$@]: Running eclint..."
 	cd $(PROJECT_ROOT) && \
 	[ -z "$(HAS_UNTRACKED_CHANGES)" ] || (echo "untracked changes detected!" && exit 1)
-	cd $(PROJECT_ROOT) && \
-	$(ECLINT_FILES) | grep -zv ".bats" | xargs -0 -I {} eclint check {}
+	eclint check $(ECLINT_FILES)
 	@ echo "[$@]: Project PASSED eclint test!"
 
-python/%: PYTHON_FILES ?= git ls-files --cached --others --exclude-standard '*.py'
+python/%: PYTHON_FILES ?= $(shell git -C $(PROJECT_ROOT) ls-files --cached --others --exclude-standard '*.py' | xargs printf "$(PROJECT_ROOT)/%s ")
```

Comment:
curious about rationale for using `shell` at this point, instead of running the command within the make target? question goes for both the python/ and eclint/ targets...

Reply:
If you're okay with the user providing a command that would we could use as input to a pipe versus a list of files, I can change it back.  Otherwise, this accommodates the fact that we need to pipe a series of commands to derive the list of files and allowing the users to simply give us a list of files.

Reply:
The Makefile started out with a 'find' command, which I changed to a 'git' command ... but really, why wasn't it just a *.py target?

Reply:
I suppose I have only a weak opinion on this one. The downsides of using `shell` is that errors seem to get masked, and you don't see the command being run, both of which can make it hard to troubleshoot. I try to avoid it if I can, but obviously we use it in many places.

If you want me to give you a firm answer now, I can play with what you have some to see if I feel more strongly either way afterwards. Otherwise, I was just curious and this is fine.

Reply:
It comes down to how you think users will derive or want to derive the list of files.  Previously, eclint was requiring the user to specify a pipe.  I think it was the same for black.  When I was testing, that was what I was doing.  It seemed cumbersome and wasn't what you typically do to override variables like this.

But I don't have a strong opinion.  I know how to change the code to go back to the previous piped method and then it's just a matter of updating the tests.

My other task for tardigrades-module is done except for bumping the version for this release of tardigrade-ci.  I'm going to start looking at the eclint replacement.

Reply:
Ok, that makes sense, if this was easier for you with your user hat on, then let's keep it like this.